docs/source/faq.rst

```diff
@@ -21,6 +21,16 @@ Is gabbi only for testing Python-based APIs?
 No, you can use :doc:`gabbi-run <runner>` to test an HTTP service
 built in any programming language.
 
+Workarounds
+~~~~~~~~~~~
+
+pytest produces warnings about yield tests. Can I make them stop?
```

Comment:
The issue is actually in the next header. rst set header levels by what it finds first, so the rest of the document indicates that === is h1 ~~~~ is h2 and --------- is h3, so the header starting 'pytest' is wrong. If you're going to fix the other thing, if you could fix this, that would be great.